src/components/Advertising.tsx

```diff
@@ -35,42 +33,25 @@ const Advertising: React.FC = () => {
   const fontSizes = [1, 2, 3, 3];
 
   return (
-    <Flex
-      flex="auto"
-      flexDirection={["column", "column", "column", "row"]}
-      justifyContent="center"
-      mt={4}
-    >
-      <Flex
-        flex="auto"
-        flexDirection="column"
-        minWidth={columnWidths}
-        width={columnWidths}
-        mr={[0, 0, 0, 2]}
-        mb={4}
-      >
-        <Img src={salazrakiImg} alt="Advertsing image" mb={2}></Img>
-        <P fontSize={fontSizes} textAlign="justify">
-          {t("advertising.salazraki")}
-        </P>
-      </Flex>
-      <Flex
-        flex="auto"
-        flexDirection="column"
-        p={0}
-        minWidth={columnWidths}
-        width={columnWidths}
-        ml={[0, 0, 0, 2]}
-        mb={4}
-      >
-        <Img src={voltrovaImg} alt="Advertsing image" mb={2}></Img>
-        <P fontSize={fontSizes} textAlign="justify">
-          {t("advertising.voltrova")}
-        </P>
-        <Img src={honestCouponsImg} alt="Advertsing image" mb={2} mt={4}></Img>
-        <P fontSize={fontSizes} textAlign="justify">
-          {t("advertising.honest-coupons")}
-        </P>
+    <Flex flex="auto" flexDirection="column">
+      <Flex flexDirection={["column", "column", "column", "row"]}>
+        <Flex flexDirection="column" width={columnWidths} mr={[0, 0, 2]}>
+          <Img src={voltrovaImg} alt="Advertsing image"></Img>
+          <P fontSize={fontSizes} textAlign="justify" mt={4} mb={6}>
+            {t("advertising.salazraki")}
+          </P>
+        </Flex>
+        <Flex flexDirection="column" width={columnWidths} ml={[0, 0, 2]}>
+          <Img src={salazrakiImg} alt="Advertsing image"></Img>
```

Comment:
this can be self closing, i know its not a change you introduced but just spotted

Reply:
eagle eyes